docs/writing-policies/go/01-intro-go.md

```diff
@@ -39,17 +39,19 @@ the actual request received by Kubernetes.
 Despite TinyGo's current limitations, it's still easy and doable to write
 Kubewarden validation policies with it.
 
-> **Note well:** unfortunately, it's currently impossible to write mutating
-> policies using TinyGo.
-
 ## Tooling
 
-Writing Kubewarden policies requires a version of TinyGo greater than `0.17.0`.
+Writing Kubewarden policies requires a version of TinyGo greater than `0.23.0`.
 
 These Go libraries are extremely useful when writing a Kubewarden policy:
 
 * [Kubewarden Go SDK](https://github.com/kubewarden/policy-sdk-go): provides a series of
   structures and functions that reduce the amount of code to write. It also provides test helpers.
+* [Kubernetes Go types](https://github.com/kubewarden/k8s-objects): the official
+  Kubernetes Go Types cannot be used with TinyGo. This module provides all the
```

Comment:
Can we link to these Go types?
```suggestion
  Kubernetes Go Types cannot be used with TinyGo. This module provides all the
```

Reply:
I honestly don't know where the code can be found, they are not inside of a dedicated repository AFAIK, you can find them [here](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io) but I don't know if this is the right location.

@ereslibre / @raulcabello do you have better ideas about what we might want to link to?

Reply:
I've used that link